web/app/js/components/util/MetricUtils.js

```diff
@@ -26,6 +26,17 @@ const convertLatencyTs = rawTs => {
   return _.groupBy(latencies, 'label');
 };
 
+export const processDeployments = pods => {
```

Comment:
thanks for adding this here! we should delete the copy of this code in `Deployments.jsx` and use this function

Reply:
also maybe we should make the name of this function more descriptive like, getPodsByDeployment or podsByDeploy....